.pylintrc

```diff
@@ -27,7 +27,9 @@ disable =
     too-many-locals,
     # Detection seems buggy or unhelpful
     duplicate-code,
-
+    # Garage env wrappers access gym private variables
+    protected-access,
```

Comment:
please disable these individually where they occur. this is a serious violation elsewhere in the codebase so we don't want to disable it everywhere.